### ISSUE TITLE
MAP search doesn't return for Contributor = "windows'd" 

### ISSUE THREAD

**Issue by Andrew-Clement** [ixmaps/website2017#86]
![pastedgraphic-1](https://cloud.githubusercontent.com/assets/12848029/25355267/dd227fc0-28ea-11e7-8510-595759891e01.png)
![pastedgraphic-2](https://cloud.githubusercontent.com/assets/12848029/25355438/8e56d55c-28eb-11e7-9c98-4b5d322352f6.png)

See: From: <user890234@tutanota.com>
Subject: Re: IXmaps client and windows firewall
Date: 22 April, 2017 4:31:46 PM PDT
To: Andrew Clement <andrew.clement@utoronto.ca>

>If you could point us to some of the routes you contributed, we could investigate further. 

I use **windows'd** as my contributor name. I have routes from home (76.201.56.0) and some from the public library.

**Comment by agamba** [comment]
Yes, the apostrophe (') is breaking the search. This needs to be dealt with in the server side
